modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldTypeSettingsTest.java

```diff
@@ -110,6 +110,17 @@ public void testCreateNumericDDMFormFieldTypeSettingsDDMForm() {
 		Assert.assertEquals(
 			"true", hideFieldDDMFormField.getProperty("showAsSwitcher"));
 
+		DDMFormField htmlAutocompleteAttribute = ddmFormFieldsMap.get(
+			"htmlAutocompleteAttribute");
+
+		Assert.assertNotNull(htmlAutocompleteAttribute);
+		Assert.assertNotNull(htmlAutocompleteAttribute.getLabel());
+		Assert.assertNotNull(
+			htmlAutocompleteAttribute.getProperty("invalidCharacters"));
+		Assert.assertEquals(
+			"true", htmlAutocompleteAttribute.getProperty("visualProperty"));
```

Comment:
Can we change it to `Assert.assertTrue()` ?
Please test if it would work.

Reply:
Done